src/assets/sass/variables/_typography.scss

```
@@ -18,10 +18,16 @@ $h1-size: 96px;
 $h1-height: 104px;
 $h1-spacing: $ls-tight;
 
+$h1-size-display: 140px;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)